site/styles/_variables.scss

```
@@ -55,11 +60,18 @@ $body-color: #fff !default;
 $body-bg: #0f0f0f !default;
 $body-bg-accent: get-accent-color($body-bg) !default;
 $error-color: $body-color !default;
-$error-background: rgba($body-color, 0.1) !default;
+$error-background: $grey-100 !default;
 $hr-border-color: rgba($body-color, 0.5) !default;
 $list-group-bg: $body-bg !default;
 $list-group-border-color: rgba($body-color, 0.125) !default;
 
+// Text Colors
+$txt-high-emphasis: $body-color;
+$txt-low-emphasis: rgba($body-color, .8);
+$txt-disabled: rgba($body-color, .6);
```

Suggest we don't need to save a character and make them all `$text-`.

```suggestion
$text-high-emphasis: $body-color;
$text-low-emphasis: rgba($body-color, .8);
$text-disabled: rgba($body-color, .6);
```

---

@PeculiarGoat FYI in case of `rgba()` goings-on.

---

Also worth noting:  CSS variables can't be passed through Sass colour manipulation functions.